components/inputs/input-time-range.js

```diff
@@ -103,6 +103,11 @@ class InputTimeRange extends RtlMixin(LocalizeCoreElement(LitElement)) {
 	render() {
 		const startLabel = this.startLabel ? this.startLabel : this.localize('components.input-time-range.startTime');
 		const endLabel = this.endLabel ? this.endLabel : this.localize('components.input-time-range.endTime');
+
+		/**
+		 * @type {'five'|'ten'|'fifteen'|'twenty'|'thirty'|'sixty'}
+		 */
```

Comment:
Yeah in other places I've created something like a `_getTimeInterval()` that has a `switch` statement and that seems to work as well. Lots of code for no reason though.